doc/stolonctl.md

```diff
@@ -1,10 +1,15 @@
 ## Stolon Client (stolonctl)
 
-`stolonctl` is the stolon client which controls the stolon cluster(s)
+`stolonctl` is the stolon client which controls the stolon cluster(s).
 
-It needs to communicate with the cluster store (providing `--store-backend` and related option) on which the requested cluster name (`--cluster-name`) is running.
+Since `stolonctl`needs to communicate with the cluster store, it requires providing the requested cluster name (`--cluster-name`), its store backend (`--store-backend`), and the kind of kubernetes resources (`--kube-resource-kind`).
 
-To avoid repeating for every command (or inside scripts) all the options you can export them as environment variables. Their name will be the same as the option name converted in uppercase, with `_` replacing `-` and prefixed with `STOLONCTL_`.
+Example of command for a cluster named "red-stolon" using "kubernetes" as a store and "configmap" as the resources:
+```
+stolonctl status --cluster-name=red-stolon --store-backend=kubernetes --kube-resource-kind=configmap
+```
+
```

Comment:
Can you also add an example for other stores (for example for an etcdv3 store)? To not make it more generic since stolon is not k8s only (and sometimes external stores are also preferred)